kalite/coachreports/static/js/coachreports/jquery.cookie.js

```diff
@@ -0,0 +1,97 @@
+/**
```

Comment:
Libraries should go under `static-libraries`, not in a project. Also, I don't see it getting used...?